packages/orbit-components/src/NotificationBadge/NotificationBadge.stories.tsx

```diff
@@ -186,3 +186,30 @@ Rtl.story = {
     info: "This is a preview of this component in RTL setup.",
   },
 };
+
+export const VisualTest = () => {
+  const Icon = getIcon(getIcons("Airplane"));
+  const types = Object.values(TYPE_OPTIONS);
+
+  return (
+    <div className="space-x-xs inline-flex">
+      {types.map(type => (
+        <React.Fragment key={type}>
+          <NotificationBadge type={type}>1</NotificationBadge>
+          <NotificationBadge type={type}>10</NotificationBadge>
+          <NotificationBadge type={type} icon={<Icon />}>
+            10
```

Comment:
does this `10` have an effect here?

Reply:
I added it to check it with two chars. Because the padding and width overrides are there to avoid the "expansion" of the badge as the content grows, so I wanted to test it with one char and two chars